cpu/stm32_common/ldscripts/stm32_common.ld

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_cortexm_common
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the Cortex-M family
+ *
+ * @author          Francisco Acosta <francisco.acosta@inria.fr>
+ *
+ * @}
+ */
+
+ccmram_length = DEFINED( ccmram_len ) ? ccmram_len : 0x0 ;
```

Comment:
I can't find any definition of `ccmram_len`, why is the purpose of that file?

Reply:
I wanted to remove it but some people convinced me to leave it as a reference. Only certain models have this kind of memory but currently is not used by our main cortex linker script. I don't think it harms and it reminds that a special memory i's available for those models, maybe in the future we'll find an usage.

Reply:
Oh, I forgot to answer the main question... I'll add the memory definition to such models.